orderer/common/localconfig/config.go

```diff
@@ -294,13 +300,28 @@ func Load() (*TopLevel, error) {
 		return nil, fmt.Errorf("Error reading configuration: %s", err)
 	}
 
-	var uconf TopLevel
-	if err := viperutil.EnhancedExactUnmarshal(config, &uconf); err != nil {
-		return nil, fmt.Errorf("Error unmarshaling config into struct: %s", err)
+	uconf := &TopLevel{}
+	serializedConf, ok := configCache[config.ConfigFileUsed()]
+	if !ok || !enableCache {
+		err := viperutil.EnhancedExactUnmarshal(config, uconf)
+		if err != nil {
+			return nil, fmt.Errorf("Error unmarshaling config into struct: %s", err)
+		}
+
+		serialized, err := json.Marshal(uconf)
+		if err != nil {
+			return nil, err
+		}
+		configCache[config.ConfigFileUsed()] = serialized
+	} else {
+		err := json.Unmarshal(serializedConf, uconf)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Same as above; it would be cool if the block above set serializedConf and this was mainline.

Reply:
done